app/managers.py

```diff
@@ -0,0 +1,45 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
```

Comment:
Using f-strings for SQL queries can lead to SQL injection vulnerabilities. It's safer to use parameterized queries, which you are already doing for the values. Consider removing the f-string and using a parameter for the table name if possible.